doc/images/d_datum_processing_one_to_one.svg

```diff
@@ -0,0 +1,2 @@
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```

Comment:
Not sure where to comment on an image so I'll just do it here. 

First, these look great! I have three very minor tweaks. 

1. the colored squares inside the black "output datums" box should be file-shaped IMO. for example in the many to many ones, it's more important to clarify that they are "file 1" and "file 2" in the output datum that then get merged. I think just change those boxes to file-shapes is simple enough.
2. It should be "output commit" not "final commit" for the last section.
3. for the 1-to-1 overwrite image, remove adding datum 4 and just make datum 3' yellow. That'll make it consistent with 1-to-many images and I think is simpler.